content/kit/wiring.md

```diff
@@ -1,3 +1,14 @@
 ---
 title: Wiring
 ---
+
+Ensuring your robot is wired correctly is incredibly important. If wired incorrectly, your kit may not work correctly, or may be damaged.
+
+## Wire Colouring
+Ground should be Black, and nothing else should be black. This is in the interest of both safety and organisation. Being able to see where all your ground cables allows you to easily track cable runs and diagnose electronic issues. It also greatly reduces the chance of running an actual current down the ground, which will serious damage the kit!
```

Comment:
"Ground should be **Black**, and nothing else should be black. " could change to lowercase.
"...all your ground cables  **go** allows you to easily track cable runs.." 
"which will serious**ly** damage the kit!"